### PR TITLE
Update prepare-code-for-review.md

### PR DESCRIPTION
**Pull request by danrabbit**
Add a link/reminder to the code style guide and a couple of other little things to check when submitting a branch

### CODE REVIEW

File: development/prepare-code-for-review.md [hunk 1]
```diff
@@ -12,7 +12,13 @@ If it's not obvious, it can also be helpful to include instructions on how to te
 
 ## Review Your Own Branch
 
-When you have your initial version for the proposed change working, look at your changes once again and ask yourself if anything could be improved before marking it "Ready for Review".
+When you have your initial version for the proposed change working, look at your changes once again and ask yourself if anything could be improved before marking it "Ready for Review". Be sure to check that:
+
+✅️ Your branch adheres to the elementary [code style guide](https://docs.elementary.io/develop/writing-apps/code-style)
+
+✅️ New Terminal warnings are not introduced
```
Comment (tintou) [inline development/prepare-code-for-review.md:19]: I'm not technically the right person to suggest that, but I had to re-read the sentence as I missed the `not` while reading, so I propose something like:
```suggestion
✅️ No New Terminal warnings are introduced
```

Comment (danrabbit) [reply development/prepare-code-for-review.md]: Makes sense to me :)

File: development/prepare-code-for-review.md [hunk 2]
```diff
@@ -12,7 +12,13 @@ If it's not obvious, it can also be helpful to include instructions on how to te
...
 ## Review Your Own Branch
 
-When you have your initial version for the proposed change working, look at your changes once again and ask yourself if anything could be improved before marking it "Ready for Review".
+When you have your initial version for the proposed change working, look at your changes once again and ask yourself if anything could be improved before marking it "Ready for Review". Be sure to check that:
+
+✅️ Your branch adheres to the elementary [code style guide](https://docs.elementary.io/develop/writing-apps/code-style)
+
+✅️ No New Terminal warnings are introduced
+
+✅️ Your branch passes CI
```
Comment (jeremypw) [inline development/prepare-code-for-review.md:21]: Is the abbreviation "CI" explained anywhere (before this page is read)?  If not, more explanation is required or a link to an explanation.

Comment (danrabbit) [reply development/prepare-code-for-review.md]: Updated this to "status checks" to be consistent with the language GitHub uses and linked to their docs

**Review by jeremypw — APPROVED** [review]

**Pull request merged**